m2e-maven-runtime/org.eclipse.m2e.maven.indexer/pom.xml

```diff
@@ -19,6 +19,7 @@
   </parent>
 
   <artifactId>org.eclipse.m2e.maven.indexer</artifactId>
+  <version>1.17.1-SNAPSHOT</version>
```

Comment:
1.17.0 has not been released, why is this necessary?

Reply:
1.17.0 has been tagged and published, and then, I found this issue to fix...